src/services/identity/ReposService.ts

```diff
@@ -219,58 +214,37 @@ export default class ReposService {
     fs.rmSync(`${dir}`, { recursive: true, force: true })
 
     // Clone base repo locally
-    await git.clone({
-      fs,
-      http,
-      dir,
-      ref: "staging",
-      singleBranch: true,
-      url: SITE_CREATION_BASE_REPO_URL,
-      depth: 1,
-    })
+    fs.mkdirSync(dir)
+    await this.simpleGit
+      .cwd(dir)
+      .clone(SITE_CREATION_BASE_REPO_URL, dir, ["-b", "staging"])
```

Comment:
sanity check here, this appears to create a branch called `staging` - will this fail if `staging` already exists? or will the `staging` branch always be created here

Reply:
this clones from the branch staging of the remote `SITE_CREATION_BASE_REPO_URL`